domains/alen.json

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "Alen's page",
+    "repo": "https://github.com/alenseeman/dev.github.io",
+    "owner": {
+        "username": "alenseeman",
+        "email": "alenseeman@hotmail.com"
+    },
+    "record": {
+        "CNAME": "dev.github.io"
```

Comment:
This doesn't match your username. 404 Site. Please send over a preview. Reopen this PR when you have this resolved.